toPrint.json

```diff
@@ -1,7 +1,7 @@
 {
-  "check_disk": 3,
-  "check_load": 5,
-  "check_size": 4,
-  "check_apt": 6,
-  "check_tcp": 1
+  "check_disk": 1,
+  "check_load": 2,
+  "check_size": 3,
+  "check_apt": 1,
+  "check_tcp": 2
```

Comment:
What is this change for?

Reply:
it was changed for test purposes should i undo?

Reply:
yes

Reply:
*bump*